src/lib/components/Swap/RoutingDiagram/utils.ts

```diff
@@ -0,0 +1,52 @@
+import { Protocol } from '@uniswap/router-sdk'
+import { Currency, Percent, TradeType } from '@uniswap/sdk-core'
+import { Pair } from '@uniswap/v2-sdk'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { css } from 'lib/theme'
+import { InterfaceTrade } from 'state/routing/types'
+
+export interface RoutingDiagramEntry {
+  percent: Percent
+  path: [Currency, Currency, FeeAmount][]
+  protocol: Protocol
+}
+
+const V2_DEFAULT_FEE_TIER = 3000
+
+export const routerGradientCss = css`
+  @supports (-webkit-background-clip: text) and (-webkit-text-fill-color: transparent) {
+    background-image: linear-gradient(90deg, #2172e5 0%, #54e521 163.16%);
```

Comment:
In the spec, the colors of the auto router label mirror the colors of the tokens being swapped. Is this supposed to use static colors instead? Seems strange to have this one thing be colorful, when everything else in the widget uses a very limited color palette.

Reply:
i actually think this is a coincidence where the spec just happens to use ETH and MKR - i think based on past designs for tooltip we've created this mini branding for the auto router, which uses this specific gradient. Ill check with Phill though to confirm that

Reply:
confirmed in slack